components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -130,12 +130,13 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
   }
 
   // Check if remapping and if so create the appropriate remapper 
-  bool m_vert_remap_from_file = params.isParameter("vertical_remap_file");
-  bool m_horiz_remap_from_file = params.isParameter("horiz_remap_file");
-  if (m_vert_remap_from_file && m_horiz_remap_from_file) {
-    // TODO - support both horizontal and vertical remapping together.
-    EKAT_ERROR_MSG("Error! scorpio_output:: We currently do not support both vertical and horizontal remapping.");
-  }
+  // Note: We currently support three remappers
+  //   - vertical remapping from file
+  //   - horizontal remapping from file
+  //   - online remapping which is setup using the create_remapper function
+  const bool use_vertical_remap_from_file = params.isParameter("vertical_remap_file");
+  const bool use_horiz_remap_from_file = params.isParameter("horiz_remap_file");
+  const bool use_online_remapper = io_grid->name()!=fm_grid->name();  // TODO: QUESTION, Do we anticipate online remapping w/ horiz_remap_from file?
```

Comment:
Since I assume we don't support it now, I would add a check to ensure we don't simultaneously request online remap AND horiz/vertical.